integrations/airflow/tests/integration/integration.py

```diff
@@ -203,13 +212,15 @@ def check_jobs_meta():
         'name': OUT_TABLE_NAME
     }]
     assert insert_job['location'] is None
-    assert insert_job['context']['sql'] is not None
-    assert insert_job['context']['airflow.operator'] == \
-           'airflow.operators.postgres_operator.PostgresOperator'
-    assert insert_job['context']['airflow.task_info'] is not None
-    assert insert_job['context']['airflow.version'] == AIRFLOW_VERSION
-    assert insert_job['context']['marquez_airflow.version'] == MARQUEZ_AIRFLOW_VERSION
-    assert insert_job['description'] == DAG_DESCRIPTION
+    # TODO: waiting for backend
+    # assert insert_job['context']['sql'] is not None
+    # assert insert_job['description'] == DAG_DESCRIPTION
+    # TODO: no airflow context job data yet
```

Comment:
Do we need to define airflow specific facets?

Reply:
I'll create PR with this tomorrow.